packages/account/src/Components/forms/formik-confirmation-checkbox/formik-confirmation-checkbox.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Checkbox, Text } from '@deriv/components';
+import { isEmptyObject, isMobile } from '@deriv/shared';
+import { useFormikContext } from 'formik';
+
+/**
+ * Props for the confirmation checkbox component.
```

Comment:
I believe, we don’t need such detailed descriptions for every types and props.
can we keep just in short what this the purpose of this component?

Reply:
Docs are referring to the purpose of the component